runtime/common/src/lib.rs

```diff
@@ -303,6 +303,7 @@ macro_rules! create_runtime {
                 Identity: pallet_identity::{Call, Event<T>, Pallet, Storage} = 30,
                 Utility: pallet_utility::{Call, Event, Pallet, Storage} = 31,
                 Proxy: pallet_proxy::{Call, Event<T>, Pallet, Storage} = 32,
+                Contracts: pallet_contracts = 33,
```

Comment:
When using this, it means that all possible pallet components are included, right?